docs/src/main/sphinx/release/release-385.md

```diff
@@ -0,0 +1,38 @@
+# Release 385 (8 June 2022)
+
+## General
+
+* Add the `json_array` and `json_object` [JSON functions](/functions/json). ({issue}`9081`)
+* Support all types that can be cast to `varchar` as parameters for the [JSON
+  path](json-path-language). ({issue}`12682`)
+* Allow `CREATE TABLE LIKE` clause on a table from a different catalog if
+  explicitly excluding table properties. ({issue}`3171`)
+* Reduce `Exceeded limit of N open writers for partitions` errors when
+  fault-tolerant execution is enabled. ({issue}`12721`)
+
+## Delta Lake connector
+
+* Add support for the [appendOnly field](https://docs.delta.io/latest/delta-batch.html#-table-properties). ({issue}`12635`)
+* Add support for column comments when creating a table or a column. ({issue}`12455`, {issue}`12715`))
+
+## Hive connector
+
+* Allow cancelling a query on a transactional table if it is waiting for a lock. ({issue}`11798`)
+* Support compression codec configuration for Avro file writing with the
```

Comment:
Add support for selecting a compression scheme when writing Avro files via the ....